package.json

```diff
@@ -23,6 +23,7 @@
     "express-basic-auth": "^1.2.0",
     "ioredis": "^4.27.9",
     "lokijs": "^1.5.12",
+    "mem": "^8.1.1",
```

Comment:
We already have a `memoizee` library inside the project with similar functionality